src/Elmish/Hooks/Type.js

```diff
@@ -1,6 +1,6 @@
-const uuidV5 = require('uuid/v5')
+const uuid = require('uuid')
```

Comment:
Not sure what’s going on with this. `examples` seems to only work with `const uuidV5 = require('uuid/v5')` and `test` only with `const uuid = require('uuid')`. Maybe it’s better to not rely on this.